ipynb/examples/utils/executor_example.ipynb

```diff
@@ -56,7 +57,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Target Configuration"
+    "## Target Configuration\n",
+    "The target configuration it's used to describe and configure your test environment.\n",
+    "You can find more details in **examples/utils/testenv_example.ipynb**."
```

Comment:
I think it would be clearer to have the TestEnv constructor call right next to the definition of my_target_conf (or even define the dict inline in the constructor). Also, a lot of the fields that are in the tests_conf below now belong in the TestEnv's test_conf dict...